speakers/src/main/res/layout/fragment_speaker_details.xml

```diff
@@ -0,0 +1,31 @@
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Same here, can we have these Speaker Details related changes in the Speaker Detail PR itself? (I believe it's not merged yet, so you can update that PR), so that we can separate concerns in the PR

Reply:
the problem is, There are a lot of refactoring for speakers module. I have moved epoxy and DTOs to base for this module, so that team, sponsors and speakers + sponsor detail, speaker-detail and team can reuse stuff. It's better to keep it incremental. As more refactoring might come if we work on sponsors.